run_tests.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
I'm really glad I found a `run_tests.py` script that I like and is easy to drop into other apps.